inst/help/RegressionLinear.md

```diff
@@ -117,7 +117,7 @@ Coefficients:
   - Upper: The upper bound of the confidence interval.
 - Collinearity Statistics:
   - Tolerance: Inverse of the Variance Inflation Factor (VIF).
-  - VIF: Variance Inflation Factor; large values indicate multicollinearity.
+  - VIF: Variance Inflation Factor; large values indicate multicollinearity. Calculated as VIF = det(R11) * det(R22) / det(R), where R is the covariance matrix of the regression coefficients (excluding intercept), R11 is a submatrix of R of the predictor for which VIF is calculated, and V22 is a submatrix of R of the other predictors (Fox & Monette, 1992).
```

Comment:
I'd use HTML for the subscripts, like so:
```suggestion
  - VIF: Variance Inflation Factor; large values indicate multicollinearity. Calculated as VIF = det(R<sub>11</sub>) * det(<sub>R22</sub>) / det(R), where R is the covariance matrix of the regression coefficients (excluding intercept), R<sub>11</sub> is a submatrix of R of the predictor for which VIF is calculated, and V<sub>22</sub> is a submatrix of R of the other predictors (Fox & Monette, 1992).
```
If our markdown engine supports LaTeX (GitHub markdown does nowadays) we could also do $R_{11}$ and stuff.

The same goes for the other languages btw.